shiny/global-sensitivity/server.R

```diff
@@ -0,0 +1,72 @@
+library(shiny)
+library(ncdf4)
+library(ggplot2)
+
+source("helper.R")
+source("plotEnsemble.R")
+source("load_ensemble.R")
+
+# Define server logic
+server <- shinyServer(function(input, output, session) {
+    bety <- betyConnect()
+    logger.setLevel("OFF")
+    ranges <- reactiveValues(x = NULL, y = NULL)
+    
+    print("RESTART")
+    
+    # set the workflow id(s)
+    ids <- get_workflow_ids(bety, session)
+    default_workflow_id <- 1000001488    # Working default
```

Comment:
No other machine will have access to this default except BU.  Also, maybe I'm just missing it, but you don't appear to be filtering your ensembles by runtype (should be 'ensemble') or number of ensemble members (i.e. runs > 1) so as to limit the menu options to actual ensemble analyses.  Oh, and since you're doing your searching by workflow.ID, how does your code handle workflows that include multiple ensemble.IDs? (e.g. if someone ran both an ensemble analysis and sensitivity analysis within the same workflow)